### ISSUE TITLE
Newline characters not being printed correctly

### ISSUE THREAD

**Issue by solarisfire** [0xacx/chatGPT-shell-cli#108]
When chatgpt responds by giving advice on how to add a new line to some Python code, it doesn't return the correct character...

e.g.

![image](https://github.com/0xacx/chatGPT-shell-cli/assets/420952/ab1e3679-4835-428c-893f-074ca5daf080)

Compared to it's response on the web chat where it does this correctly:

![image](https://github.com/0xacx/chatGPT-shell-cli/assets/420952/41366050-b0d2-4803-b187-7c09110b9e1f)

**Comment by solarisfire** [comment]
Oh, I guess I should have looked at the latest commit before posting this... 🤦🏻

**Comment by solarisfire** [comment]
Nope, updated to latest code from git and still getting the same issue:

![image](https://github.com/0xacx/chatGPT-shell-cli/assets/420952/85590f91-d6e8-4ae5-8c28-2a393b166a15)

**Comment by mountaineerbr** [comment]
You may try to install `glow`.

It seems you haven't got `glow`, so I reckon the problem is with `echo -e` in [line 450](https://github.com/0xacx/chatGPT-shell-cli/blob/926587a2234b8ae3754a1db9715f4636205159dc/chatgpt.sh#L450).

OBS: I am not sure how to fix it because IDK how text in `$formatted_text` is stored (I would rather not install this script for testing myself as it messes up with sys configs).

**Comment by solarisfire** [comment]
Okay, that fixes it!

![image](https://github.com/0xacx/chatGPT-shell-cli/assets/420952/b2b94166-052a-4e01-aea9-1365410837d0)

Maybe glow needs to be a dependency, or there should be some warning if glow isn't installed suggesting that it should be?

**Comment by 0xacx** [comment]
Yes glow is an optional dependency and listed as optional in the [prerequisites](https://github.com/0xacx/chatGPT-shell-cli#prerequisites). Meaning this should work fine also without installing glow. I can't look into this right now so I will leave this issue open to examine.
